### ISSUE TITLE
tnslint command line execution doesn't report errors for naming-convention rule

### ISSUE THREAD

**Issue by srivelicheti** [ajafff/tslint-consistent-codestyle#140]
I see warning\squiggly in vs code for naming-convention rule violation, however if I run tslint in command line it doesn't show any errors.

![image](https://user-images.githubusercontent.com/3229242/71936777-49eacc00-315f-11ea-8710-fe8ce6934063.png)

My tslint.json
```json
{
  "extends": [],
  "rules": {
    "naming-convention": [
      true,
      {
        "type": "default",
        "format": "camelCase",
        "leadingUnderscore": "forbid",
        "trailingUnderscore": "forbid"
      },
      {
        "type": "member",
        "modifiers": "private",
        "leadingUnderscore": "require"
      }
    ]
  },
  "rulesDirectory": ["codelyzer", "tslint-consistent-codestyle"]
}

```

TSLint package version being used is 5.20.1
